### DISCUSSION TITLE
Scrum Prep Milestone #2 - Second Week

### DISCUSSION THREAD

**Discussion by YanehRuiz** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#240] [Weekly Retrospective]
Weekly Development Update: Second week of Milestone 2

Hello team!

Please provide a brief comment detailing what you accomplished this week, including any challenges faced or insights gained. Make sure to submit your updates by the deadline, which is before: Friday, October 4, 2024 @ 11am.

Thank you for your collaboration!

As for my collaboration:

**Comment by CaribbeanCool** [comment]
I haven't done much. I'm beginning to develop my task #208

**Comment by evnd04** [comment]
Ive been mostly reviewing PR's, grading issues and revising new issues daily.

**Comment by noel-vargas** [comment]
Created a draft for the technical flow of props and components of the react application (Profile Management Tab).

**Comment by javier-rivera8** [comment]
This week, I have started working on designing the deletion page to add the report history.

**Comment by Jjulianirizarry24** [comment]
Organized team meeting. Created multiple issues for team members.

**Comment by JuanJRivera** [comment]
I did not work this week, I had a week full of exams.

**Comment by johnsantiago9** [comment]
This week I wasn’t able to work due to a lot of examen, but currently looking to finish sone documentation tasks and finish the tasks I have assigned

**Comment by misaelmercado1** [comment]
Working on the design and implementation of the button for the location management team that displays the list of reported locations.

**Comment by Alejandro-DeJesus** [comment]
This week, I worked and Completed issue #215 where I added a report tab to the application.

**Comment by RaquelQuinones** [comment]
This week, I fixed the firebase issue and also completed task #178.

**Comment by jonVargas1** [comment]
This week I haven't done much but when possible I have been brainstorming ideas for new issues.

**Comment by ghost** [comment]
I found a bug in the database dummy data population script, and fixed it #238. I still have planned to finish adding more dummy data to the database #214.

**Comment by JorgeRivera94** [comment]
Due to the workload of other courses, this week I have only been able to get some progress on the reclassification of report categories.

**Comment by williammartinez10** [comment]
Started working on task #220

**Comment by lex939** [comment]
Uploaded multiple documentation issues with more to come. Will start revising and adding along with the rest of the hybrids soon.

**Comment by Rodal1** [comment]
Haven't done a lot, will begin homepage task for administrative interface.

**Comment by Chosen-Juan1** [comment]
I have not been able to accomplish much this week due to other courses. Will try to finish the admin page design for next week.

**Comment by shenied-maldonado** [comment]
It was a slow week. I cleared up some problems and miscommunications regarding some of my team's assigned tasks and worked on implementing some error handling for our CRUD operations. I'm only working on the fine details now.

**Comment by NathBat09** [comment]
Worked on Issue #243 with configuring priority fields in the database.

**Comment by yadrielsamir** [comment]
For this week I conducted the weekly meeting for admin team and finished admin calendar design.

**Comment by JorgeC08** [comment]
Didn't do much since it's been hell week but created one task, expected to finish it before Monday

**Comment by joel-alvarado** [comment]
Worked on email verification system in the Flask server side.

**Comment by lex939** [comment]
This discussion is now closed since in-person class will begin soon.
